pkg/storages/gcs/xmlupload.go

```diff
@@ -0,0 +1,243 @@
+package gcs
+
+import (
+	"bytes"
+	"context"
+	"encoding/xml"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"net/url"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/wal-g/tracelog"
+	"golang.org/x/oauth2"
+	auth "golang.org/x/oauth2/google"
+)
+
+const (
+	// reference : https://cloud.google.com/storage/quotas#requests
+	MAX_PARTS     = 10000
+	MAX_PART_SIZE = 5 << 30
+	MIN_PART_SIZE = 5 << 20
+)
+
+type InitiateUploadResponse struct {
+	XMLName  xml.Name `xml:"InitiateMultipartUploadResult"`
+	Bucket   string   `xml:"Bucket"`
+	Key      string   `xmml:"Key"`
```

Comment:
Possible typo: `xmml` instead of `xml`. 
Will result in an incorrect unmarshaling (zero string) of the XML response. 

Not used in `getUploadId`, but should still be fixed.